src/consumer/base_consumer.rs

```diff
@@ -313,6 +313,25 @@ impl<C: ConsumerContext> Consumer<C> for BaseConsumer<C> {
         }
     }
 
+    fn offsets_for_times<T: Into<Option<Duration>>>(&self, mut timestamps: TopicPartitionList, timeout: T)
```

Comment:
Would it make sense to add a comment here indicating that the `timestamps` is expected to contain have their offset set to the timestamp to search for?